test/third_party_tests/test_native_executor.py

```diff
@@ -228,6 +228,26 @@ def test_should_run_query_in_clickhouse(self):
         self._execute_native_query()
         self._execute_evadb_query()
 
+    def test_should_run_query_in_snowflake(self):
+        # Create database.
+        params = {
+            "user": "eva",
+            "password": "password",
+            "account": "account_number",
+            "database": "EVADB",
+            "schema": "SAMPLE_DATA",
+            "warehouse": "warehouse",
+        }
```

Comment:
Is this a valid connection?

Reply:
No, it is not, do we need to test it against a real snowflake account ?

Reply:
Yes. Otherwise the test will not run. Does snowflake has any free account we can test with?

Reply:
I see. Then we can skip this test. Use `@pytest.mark.skip(reason="...")`